.github/workflows/feature-tests.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
-      - name: set up JDK 1.8
+      - name: set up JDK 11
         uses: actions/setup-java@v1
         with:
-          java-version: 1.8
+          java-version: 11
       - name: creating local.properties # this file is normally added to .gitignore
```

Comment:
@bohdan-go-wombat can we replace the above with:
```yaml
- uses: actions/checkout@v4
- uses: actions/setup-java@v4
  with:
    distribution: 'zulu'
    java-version: '11'
    cache: 'gradle'
```

(and do the same on the other workflow files as well)

Reply:
@andresilveirah agree, not sure about the distribution though, on our local machines we are using Corretto distribution instead of Zulu, but maybe we should try it out

Reply:
Yeah, I looked for corretto, but the gh action doesn't have it. I think Zulu (or any other distribution should be fine).